<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportPivotTable</class>
 <widget class="QDialog" name="ImportPivotTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ignoreListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Feature columns</string>
   </property>
  </widget>
  <widget class="SingleListWidget" name="sampleListWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="SingleListWidget" name="lipidListWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Columns to ignore</string>
   </property>
  </widget>
  <widget class="QListWidget" name="featureListWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>191</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample column</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Lipid column</string>
   </property>
  </widget>
  <widget class="SingleListWidget" name="quantListWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Quant data column</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SingleListWidget</class>
   <extends>QListWidget</extends>
   <header>lipidspace/AssistanceFunctions.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
